spyder/workers/tests/test_update.py

```diff
@@ -22,13 +22,20 @@ def test_update(qtbot):
     assert check_version("1.0.0", worker.latest_release, '<')
 
 
-def test_not_update(qtbot):
+def test_no_update(qtbot):
     """Test the update checking for a version that don't needs an update."""
     worker = WorkerUpdates(None, False, version="1000.0.0")
     worker.start()
     assert not worker.update_available
     assert not check_version("1000.0.0", worker.latest_release, '<')
 
 
+def test_no_update_development(qtbot):
```

Comment:
Should beta versions be considered development versions for the purposes of the updater? Either way, shouldn't we test that they're handled properly?